packages/theme/src/ThemeProvider.tsx

```diff
@@ -14,12 +19,18 @@ export const ThemeProvider = ({children}: {children: React.ReactNode}) => {
 
   const selectColorScheme = useSaveColorScheme()
   const colorScheme = savedColorScheme ?? systemColorScheme
-  const theme = themes[isColorScheme(colorScheme) ? colorScheme : systemColorScheme]
+  const theme =
+    themes[isColorScheme(colorScheme) ? colorScheme : systemColorScheme]
 
-  return <ThemeContext.Provider value={{theme, colorScheme, selectColorScheme}}>{children}</ThemeContext.Provider>
+  return (
+    <ThemeContext.Provider value={{theme, colorScheme, selectColorScheme}}>
```

Comment:
@SorinC6 I'd say we should add 2 more attributes, `isLight` `isDark`, there are a lot of references to it, by returning directly from the context, will spare some logic in the components.